03-specifying-sample-designs.Rmd

```diff
@@ -112,7 +114,7 @@ des_apisrs
 summary(des_apisrs)
 ```
 
-In the printed design object above, the design is described as an "Independent Sampling design," which is another terminology for SRS. The ids are specified as `1`, which means there is no clustering (a topic described later in this chapter), the fpc variable is indicated, and the weights are indicated. When looking at the summary of the design object, the population size is given as a summary of the probabilities (inverse of the weights).
+In the printed design object above, the design is described as an "Independent Sampling design," which is another terminology for SRS. The ids are specified as `1`, which means there is no clustering (a topic described later in this chapter), the fpc variable is indicated, and the weights are indicated. When looking at the summary of the design object, the population size is given as a summary of the probabilities (inverse of the weights). 
```

Comment:
```suggestion
In the printed design object above, the design is described as an "Independent Sampling design," which is another term for SRS. The ids are specified as `1`, which means there is no clustering (a topic described later in this chapter), the fpc variable is indicated, and the weights are indicated. When looking at the summary of the design object, the population size is given as a summary of the probabilities (inverse of the weights). 
```

Reply:
To not have to add a space after a bunch of paragraphs, I think we should go through and capitalize FPC when appropriate.

Reply:
This paragraph should move up before the "summary" statement in the code.  If we need to break out the code chunks then we should.  We really should have a common structure throughout that is description of what we're going to do/see, then see it.  This is common with tables and figures in scientific writing and we should try to implement that throughout.  Adding this comment here, but this should be consistent throughout the book.